README.md

```diff
@@ -44,11 +44,15 @@ For example, to include only S3 and SQS:
 
 ``` toml
 [dependencies]
-rusoto_core = {version = "0.25.0"}
-rusoto_sqs = {version = "0.25.0"}
-rusoto_s3 = {version = "0.25.0"}
+rusoto_core = "0.26.0"
+rusoto_sqs = "0.26.0"
+rusoto_s3 = "0.26.0"
 ```
 
+## Migrating from Rusoto 0.24.0 to 0.25.0 or later
+
+See the [Rusoto 0.25.0 release notes](https://github.com/rusoto/rusoto/releases/tag/rusoto-v0.25.0) for more information.
```

Comment:
I think we should include a few more words up front about this - something to tell users that 0.25.0 was a major architectural change to how the crate was packaged.  Otherwise people might just skim by and not click the link.